src/main/io/gps.c

```diff
@@ -473,6 +543,92 @@ void gpsInitNmea(void)
 #endif // USE_GPS_NMEA
 
 #ifdef USE_GPS_UBLOX
+const uint8_t ubloxUTCStandardConfig_int[5] = {
+        UBLOX_UTC_STANDARD_AUTO,
+        UBLOX_UTC_STANDARD_USNO,
+        UBLOX_UTC_STANDARD_EU,
+        UBLOX_UTC_STANDARD_SU,
+        UBLOX_UTC_STANDARD_NTSC
+};
+
+struct ubloxVersion_s ubloxVersion_map[] = {
```

Comment:
```suggestion
struct ubloxVersion_s ubloxVersionMap[UBX_VERSION_COUNT] = {
```

Reply:
Done

Reply:
Not yet. `UBX_VERSION_COUNT` is missing

Reply:
Ah OK, thank you - done!

Reply:
@KarateBrot : I'm afraid that this will only catch excess initializer, not missing one.

Reply:
Please let me know what we should do, here :-)

Reply:
ledvinap says that the compiler only yells if we have too many elements in the list. The compiler will not yell if we have too little elements in the list. But that's still better than the compiler not yelling at all, IMO, so I'd still keep it this way